staging/src/k8s.io/apimachinery/pkg/util/net/util.go

```diff
@@ -47,6 +48,11 @@ func IsConnectionReset(err error) bool {
 	return false
 }
 
+// Returns if the given err is "http2: client connection lost" error.
+func IsConnectionLost(err error) bool {
+	return err != nil && strings.Contains(err.Error(), "http2: client connection lost")
```

Comment:
error is here https://cs.opensource.google/go/x/net/+/master:http2/transport.go;l=1134;drc=d28c0b17437aac95ef32d25b3ff2c919ee165ebb

Reply:
Done, thanks!